sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/ToNumberParser.scala

```diff
@@ -165,6 +166,10 @@ class ToNumberParser(numberFormat: String, errorOnFail: Boolean) extends Seriali
           tokens.prepend(OpeningAngleBracket())
           tokens.append(ClosingAngleBracket())
           i += 2
+        case SPACE =>
```

Comment:
hmm, now we allow spaces in the format pattern?

Reply:
Yeah, I added this small change because the Spark `.sql` tests specify this, and it is compatible with Postgres. I asked Serge about this to see if he has opinions.

<img width="1530" alt="image" src="https://user-images.githubusercontent.com/99207096/165815005-431ed4c6-cb4b-4813-bd84-91d690c0a2d2.png">